Makefile

```diff
@@ -8,9 +8,14 @@ corename:
 # SIMULATE
 #
 
+#sim target: to be reviewed
```

Comment:
this comment does not make sense if you are using git

Reply:
Will be removed